src/main/java/net/coderbot/patchwork/manifest/forge/AccessTransformerEntry.java

```diff
@@ -0,0 +1,49 @@
+package net.coderbot.patchwork.manifest.forge;
+
+import net.fabricmc.tinyremapper.MemberInstance;
+import org.objectweb.asm.commons.Remapper;
+
+public class AccessTransformerEntry {
+	private String clazzName;
+	private String memberName;
+	private boolean memberIsField = false;
+	// todo inner class support
+	public AccessTransformerEntry(String clazzName,
```

Comment:
I think we should have a generic constructor for a class and member name, and instead expose the remap() method as public.